docs/01_key-concepts/06_vote.md

```diff
@@ -1,6 +1,6 @@
 ---
-content_title: Voting on EOSIO based blockchains
-link_text: Voting on EOSIO based blockchains
+content_title: Voting on EOSIO-based blockchains
+link_text: Voting on EOSIO-based blockchains
 ---
 
-In a EOSIO-based network the blockchain is kept alive by nodes which are interconnected into a mesh, communicating with each other via peer to peer protocols. Some of these nodes are elected, via a __voting__ process, by the token holders to be producer nodes. They produce blocks, validate them and reach consensus on what transactions are allowed in each block, their order, and what blocks are finalized and stored forever in the blockchain memory. This way the governance, the mechanism by which collective decisions are made, of the blockchain is achieved through the 21 active block producers which are appointed by token holders' __votes__. It's the 21 active block producers which continuously create the blockchain by creating blocks, and securing them by validating them, and reaching consensus. Consensus is reached when 2/3+1 active block producers agree on validity of a block, that is all transactions contained in it and their order.
+In a EOSIO-based network the blockchain is kept alive by nodes which are interconnected into a mesh, communicating with each other via peer to peer protocols. Some of these nodes are elected, via a voting process, by the token holders to be producer nodes. They produce blocks, validate them and reach consensus on what transactions are allowed in each block, their order, and what blocks are finalized and stored forever in the blockchain memory. This way the governance, the mechanism by which collective decisions are made, of the blockchain is achieved through the 21 active block producers which are appointed by token holders' votes. It's the 21 active block producers which continuously create the blockchain by creating blocks, and securing them by validating them, and reaching consensus. Consensus is reached when 2/3+1 active block producers agree on validity of a block, that is all transactions contained in it and their order.
```

Comment:
- Doesn't have to be a `mesh`. Nodes can be connected anyway node operators wish.
- "stored forever in the blockchain memory" I would drop the word "memory".
- EOS mainnet has 21 producers, but this is configurable. Other changes can have different # of block producers up to the hard coded max producers of 125.